apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/modal.js

```diff
@@ -73,12 +88,7 @@ const GlobalStylesModal = () => {
 					<h1 className="wpcom-global-styles-modal__heading">
 						{ __( 'A powerful new way to style your site', 'full-site-editing' ) }
 					</h1>
-					<p className="wpcom-global-styles-modal__description">
-						{ __(
-							"Change all of your site's fonts, colors and more. Available on the Premium plan.",
-							'full-site-editing'
-						) }
-					</p>
+					<p className="wpcom-global-styles-modal__description">{ description }</p>
```

Comment:
I think we can select the translation here in this case, in `apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/notices.js` we have it extracted because the translation is in a `createInterpolateElement` call.

Reply:
Good call. Although I personally prefer it this way, it feels more readable. And anyway, it's temporary code after all 😄